wallet/src/main/scala/org/bitcoins/wallet/config/WalletAppConfig.scala

```diff
@@ -154,10 +154,21 @@ case class WalletAppConfig(
         migrate()
       }
 
+      if (isHikariLoggingEnabled) {
+        //.get is safe because hikari logging is enabled
+        startHikariLogger(hikariLoggingInterval.get)
+      }
```

Comment:
This is here as well as in every app config's start function, shouldn't it be one or the other? Preferably just here

Reply:
If I understand your question

every module has it's own connection pool allocated to access the database, thus we need a hikari logger to monitor the status of each connection pool.